[linking] make universal linking more prominent (#34758)
expo/expo: docs/pages/linking/into-your-app.mdx

import { Collapsible } from '~/ui/components/Collapsible';
import { Terminal } from '~/ui/components/Snippet';

> This guide provides steps to configure standard **deep links** in your project by adding a custom scheme. For production apps, you probably want to (also) set up app [Universal linking](/linking/overview/#universal-linking).

Reviewer: Techbnically, Universal Links is the iOS term and the equivalent concept on Android is called App Links. The Android [docs](https://developer.android.com/training/app-links) on the topic don't use the term "universal" at all.

```suggestion
This guide provides steps to configure standard **deep links** in your project by adding a custom scheme. For production apps, you probably want to (also) set up app [Universal / App Links](/linking/overview/#universal-linking).
```

This guide provides steps to configure standard **deep links** in your project by adding a custom scheme.

> > **info** For most apps, you probably want to set up [Android App/iOS Universal Links](/linking/overview/#universal-linking) instead of the deeps links documented here, or set up both.

Reviewer 2: ```suggestion
> **info** For most apps, you probably want to set up [Android App/iOS Universal Links](/linking/overview/#universal-linking) instead of the deep links described in this guide or set up both.
```
